docs/fiddles/features/drag-and-drop/preload.js

```diff
@@ -0,0 +1,12 @@
+const { contextBridge, ipcRenderer } = require('electron')
+const fs = require('fs').promises
+const path = require('path')
+
+contextBridge.exposeInMainWorld('electron', {
+  startDrag: async (fileName) => {
+    // Create a new file to copy - you can also copy existing files.
+    await fs.writeFile(fileName, '# Test drag and drop')
```

Comment:
nit: I don't think we should be creating the file every time we start the drag-and-drop. Would it make sense to put this line in the main process?

Reply:
Yeah, this seemed weird to me. I'll change it.